src/components/Column/index.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
+
+import "./index.scss";
+
+type Width = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12;
+type Offset = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11;
+
+interface ColumnProps {
```

Comment:
Nice bringing in the Column and Row components and glad they're working okay for yall 😄 . When i said they were written in TypeScript you guys didn't have to bring in TypeScript and change your webpack config to add TypeScript but you guys could've made it in normal JSX class components. we'll use mostly proptypes and JSX files for now as this is a prelude to eventually converting everything over to TypeScript so you guys are already a step ahead 🥇  and we wanted you guys to get used to typing in general before throwing the whole TypeScript at yall . I'm okay with you guys leaving in the index.tsx and the outputted JS files from TypeScript for these Row/Column components since i feel it helps breakdown how you guys will use it, but for the rest of the components let's continue to use normal .jsx files and we'll get to TypeScript eventually after MVP 👍